pnpm-lock.yaml

```diff
@@ -1,9 +1,5 @@
 lockfileVersion: '6.0'
 
-settings:
```

Comment:
this is worrying me

Reply:
looks like it was moved to the end of this file for some reason. lgtm